### PR TITLE
Changed duration of animation "How it works" block to 0.3s

### PR DESCRIPTION
**Pull request by ArturBekhDEV**

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1,2 @@
+REACT_APP_API_BASE_PATH=http://localhost:8080
```
Comment (MickhaelSh) [inline .env:1]: file should be called .env.local

File: src/containers/app-content/AppContent.js [hunk 2]
```diff
@@ -14,7 +14,6 @@ const styles = {
     height: '100vh'
   }
 }
-
```
Comment (MickhaelSh) [inline src/containers/app-content/AppContent.js:17]: why you removed this empty line?

File: src/styles/app-theme/custom-animations.js [hunk 3]
```diff
@@ -36,9 +36,9 @@ export const fadeAnimation = {
 }
 
 export const slidesRightAnimation = {
```
Comment (MickhaelSh) [inline src/styles/app-theme/custom-animations.js:38]: Change transition time in component too

File: .env (hunk 1 above)
Comment (ArturBekhDEV) [reply .env]: fixed

File: src/containers/app-content/AppContent.js (hunk 2 above)
Comment (ArturBekhDEV) [reply src/containers/app-content/AppContent.js]: fixed

File: src/styles/app-theme/custom-animations.js (hunk 3 above)
Comment (ArturBekhDEV) [reply src/styles/app-theme/custom-animations.js:38]: fixed

**Review by MickhaelSh — APPROVED** [review]

**Review by Tolik170 — APPROVED** [review]

**Review by Radomyr-kh — APPROVED** [review]

**Review by Yurenko — APPROVED** [review]

**Review by KhrystynaPavlikovska — APPROVED** [review]

**Review by maarekua — APPROVED** [review]

**Pull request merged**